src/config.hh

```diff
@@ -455,24 +455,26 @@ struct MediaWiki
   QString id, name, url;
   bool enabled;
   QString icon;
+  QString lang;
 
   MediaWiki():
     enabled( false )
   {
   }
 
-  MediaWiki( QString const & id_, QString const & name_, QString const & url_, bool enabled_, QString const & icon_ ):
+  MediaWiki( QString const & id_, QString const & name_, QString const & url_, bool enabled_, QString const & icon_, QString const & lang_ ):
```

Comment:
QString const & lang_={}

add a default value. then all the invoked codes can be avoid changes.

Reply:
fixed.